geom/alg_densify_test.go

```diff
@@ -0,0 +1,75 @@
+package geom_test
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/peterstace/simplefeatures/geom"
+)
+
+func TestDensifyEmpty(t *testing.T) {
+	for _, empty := range []geom.Geometry{
+		geom.Point{}.AsGeometry(),
+		geom.LineString{}.AsGeometry(),
+		geom.Polygon{}.AsGeometry(),
+		geom.MultiPoint{}.AsGeometry(),
+		geom.MultiLineString{}.AsGeometry(),
+		geom.MultiPolygon{}.AsGeometry(),
+		geom.GeometryCollection{}.AsGeometry(),
+	} {
+		t.Run(empty.String(), func(t *testing.T) {
+			for _, ct := range []geom.CoordinatesType{
+				geom.DimXY,
+				geom.DimXYZ,
+				geom.DimXYM,
+				geom.DimXYZM,
+			} {
+				t.Run(ct.String(), func(t *testing.T) {
+					input := empty.ForceCoordinatesType(ct)
+					got := input.Densify(1.0)
+					expectGeomEq(t, got, input)
+				})
+			}
+		})
+	}
+}
+
+func TestDensify(t *testing.T) {
+	for i, tc := range []struct {
+		input   string
+		minDist float64
```

Comment:
Should this be `maxDist`?

Reply:
Yes! Fixed in 557a096cf6b2ea30bd306128146fd81ac2f03cad